pipelines/batch/src/main/java/com/google/fhir/analytics/BulkExportApiClient.java

```diff
@@ -125,20 +130,29 @@ public BulkExportHttpResponse fetchBulkExportHttpResponse(String bulkExportStatu
   }
 
   private IBaseParameters fetchBulkExportParameters(
-      FhirVersionEnum fhirVersionEnum, List<String> resourceTypes) {
+      FhirVersionEnum fhirVersionEnum, List<String> resourceTypes, @Nullable String since) {
     switch (fhirVersionEnum) {
       case R4:
         Parameters r4Parameters = new Parameters();
         r4Parameters.addParameter(PARAMETER_TYPE, String.join(",", resourceTypes));
+        if (!Strings.isNullOrEmpty(since)) {
+          r4Parameters.addParameter(PARAMETER_SINCE, new InstantType(since));
+        }
         return r4Parameters;
       case DSTU3:
         org.hl7.fhir.dstu3.model.Parameters dstu3Parameters =
             new org.hl7.fhir.dstu3.model.Parameters();
-        ParametersParameterComponent parametersParameterComponent =
-            new ParametersParameterComponent();
-        parametersParameterComponent.setName(PARAMETER_TYPE);
-        parametersParameterComponent.setValue(new StringType(String.join(",", resourceTypes)));
-        dstu3Parameters.addParameter(parametersParameterComponent);
+        ParametersParameterComponent typeParameter = new ParametersParameterComponent();
```

Comment:
It is sad that there is no `Parameter` manipulation interface and we have to duplicate this code for each FHIR version. Not for this PR, but consider adding a TODO to define a common interface for adding parameters.